modules/meb_api/app/controllers/meb_api/v0/education_benefits_controller.rb

```diff
@@ -3,58 +3,84 @@
 module MebApi
   module V0
     class EducationBenefitsController < MebApi::V0::BaseController
+      # disabling checks while we serve big mock JSON objects. Check will be reinstated when we integrate with DGIB
+      # rubocop:disable all
```

Comment:
Can you inline disable the specific rubocop error here instead of a disable all?

Reply:
Will address.

Reply:
I've added   # rubocop:disable Metrics/MethodLength instead of disable all. I think we are good now.